src/webportal/src/app/job-submission/components/data/add-hdfs.jsx

```diff
@@ -134,34 +131,33 @@ export const AddHDFS = ({
             }
           }}
         />
-      </div>
-      <div>
-        <Stack horizontal>
-          <div className={c(FontClassNames.smallPlus)}>
-            The path in PAI HDFS
-          </div>
-          <div className={c(t.red, t.pl1, t.mb2)}>*</div>
-        </Stack>
+      </Stack.Item>
+      <Stack.Item align='baseline'>
+        <Label required className={FontClassNames.medium}>
+          Path in pai HDFS
+        </Label>
         <TagPicker
+          disabled={hdfsPathErrorMessage === 'Pai HDFS is not available'}
           onResolveSuggestions={onFilterChanged}
           onItemSelected={onItemSelected}
           pickerSuggestionsProps={{
-            suggestionsHeaderText: 'path in hdfs should start with /',
-            noResultsFoundText: 'path not found',
+            suggestionsHeaderText: 'Path in hdfs should start with /',
+            noResultsFoundText: 'Path not found',
           }}
           itemLimit={1}
-          styles={{
-            root: {
-              width: 200,
-              marginBottom: containerPathErrorMessage
-                ? hdfsPathErrorMessage
-                  ? 0
-                  : 22.15
-                : 0,
-            },
-          }}
         />
-      </div>
+        {hdfsPathErrorMessage && (
+          <span
+            className={FontClassNames.small}
+            style={{
+              color: palette.redDark,
```

Comment:
use `semanticColors.errorText` for the color?

Reply:
fixed